### PR TITLE
testing/netcdf: new aport

### PR DESCRIPTION
**Pull request by hjaekel**

### CODE REVIEW

File: testing/netcdf/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+utils() {
+	install -d "$subpkgdir/usr/bin"
+	mv "$pkgdir/usr/bin" "$subpkgdir/usr/"
+}
+
+check() {
+	# One test failure https://github.com/Unidata/netcdf-c/issues/808
+	ctest || warning "Test failure in nc_test as reported in https://github.com/Unidata/netcdf-c/issues/808"
```
Comment (maxice8) [inline testing/netcdf/APKBUILD]: you can do `-E testname` to make the test not run

**Comment by hjaekel** [comment]
I added `-E nc_test`. Thank you for the review.

**Comment by hjaekel** [comment]
After disabling `nc_test` there was still a test failure on armhf and armv7 where the program under test crashed. So I disabled those platforms.

File: testing/netcdf/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Contributor: Holger Jaekel <holger.jaekel@gmx.de>
+# Maintainer: Holger Jaekel <holger.jaekel@gmx.de>
+pkgname=netcdf
+pkgver=4.7.0
+pkgrel=0
+pkgdesc="Libraries for the Unidata network Common Data Format"
+url="https://www.unidata.ucar.edu/software/netcdf/"
+arch="x86_64 x86 aarch64 ppc64le s390x"  # disabling on armhf and armv7 as tests fail
```
Comment (maxice8) [inline testing/netcdf/APKBUILD]: you can do arch="all !armhf !armv7"

Comment (hjaekel) [reply testing/netcdf/APKBUILD]: > you can do `arch="all !armhf !armv7"`

done

**Pull request closed without merging**